### ISSUE TITLE
wayland/sway/waybar: no icon in system tray, gtk errors in log

### ISSUE THREAD

**Issue by telychko** [valr/cbatticon#74]
Everything is working OK except tray icon.

Environment: 
* sway version 1.9
*  Waybar v0.10.3

$ cbatticon --version
cbatticon: a lightweight and fast battery icon that sits in your system tray
version 1.6.13

Command to run:
`cbatticon --low-level 30 --critical-level 25`

Errors from the log
(cbatticon:982): Gtk-CRITICAL **: 23:10:41.370: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed

**Comment by Tommimon** [comment]
Same issue here, same error in the log. Did you find a solution?

**Comment by telychko** [comment]
> Same issue here, same error in the log. Did you find a solution?

Nope. At the moment, I'm using a popup notifications functionality only

**Comment by artist4xlibre** [comment]
This pull request has been merged into https://github.com/artist4xlibre/batticonplus/releases/tag/v2.0.0
